src/Microsoft.Data/PrimitiveDataFrameColumn.Sort.cs

```diff
@@ -33,7 +33,7 @@ private void GetSortIndices(IComparer<T> comparer, out PrimitiveDataFrameColumn<
             for (int b = 0; b < _columnContainer.Buffers.Count; b++)
             {
                 ReadOnlyDataFrameBuffer<T> buffer = _columnContainer.Buffers[b];
-                ReadOnlySpan<byte> nullBitMapSpan = _columnContainer.NullBitMapBuffers[b].ReadOnlySpan;
+                ReadOnlySpan<byte> nullBitMapSpan = _columnContainer.NullBitMapBuffers[b].RawReadOnlySpan;
```

Comment:
Each `NullBitMapBuffer` stores flags for 8 indices in 1 byte. However, `NullBitMapBuffer.Length` is set to numberOfBytes * 8 to match it's data buffer's `Length`. `NullBitMapBuffer.ReadOnlySpan` internally calls `Slice(start, Length)`, but that could fail with an `IndexOutOfRangeException`. Instead, when dealing with NullBitMapBuffers, `NullBitMapBuffer.RawReadOnlySpan` should be used to get the span of bytes. I missed 1 spot that still used `ReadOnlySpan`.